cmd/pebble/cmd_run.go

```diff
@@ -49,6 +51,7 @@ func init() {
 	addCommand("run", shortRunHelp, longRunHelp, func() flags.Commander { return &cmdRun{} },
 		map[string]string{
 			"create-dirs": "Create pebble directory on startup if it doesn't exist",
+			"dry":         "Don't start the pebble daemon. This option is incompatible with --create-dirs",
```

Comment:
I'd prefer a help message that was a bit more direct ("Does X" rather than "Don't ..."). Perhaps "Dry run: validate plan without starting daemon"? Or would the argument better be called `--validate` or `--validate-only`, as it seems that's the intent, not really a dry *run*?

If it is "dry run", why is it incompatible (only) with `--create-dirs`? It seems like it's also incompatible with `--hold` and `--http` and `--verbose`, but we don't check for those. IMO we should just drop this check.